core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -3437,6 +3437,9 @@ private void rewriteNegativeLimit(QueryModel model, SqlExecutionContext executio
                 int timestampKey = current.getColumnNameToAliasMap().keyIndex(nested.getTimestamp().token);
                 if (timestampKey < 0) {
                     int timestampIdx = current.getColumnAliasIndex(current.getColumnNameToAliasMap().valueAt(timestampKey));
+                    if (timestampIdx == -1) {
```

Comment:
whoooooops left in there by mistake?

Reply:
removed here https://github.com/questdb/questdb/pull/3684/commits/e62df8b72ce09273453bee6c5ebdeeb5f775f801